github-issue-classification/k8s/manifests/train-pod.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: "v1"
```

Comment:
The idea behind the GIC usecase is that we train the model in a distributed fashion using Kubeflow TFJobs; the manifest for creating that is already in this repo. I'd like to get a bit more context on how you propose these new pods to be ran.

Reply:
I have added this yaml to support GIC on a plain kubernetes cluster without kubeflow.

Reply:
Hey @mak-454 thanks for the reply. I get it now. Some requests:
* Let's create new manifests for kubernetes only deployments and keep kubeflow manifests as they are right now (the one that shouldn't change at all is tf_job-github.yaml). 
* Currently all pods are created on the kubeflow namespace, that's something that must be changed either specifying on which namespace they will be created using the CLI or right in the manifest. I'll leave that up to you. If we use the CLI we can re use some manifests, though. 
* Let's create a new section on the readme specifying how to run this using only kubernetes.